### DISCUSSION TITLE
Cannot see Nameservers Tab

### DISCUSSION THREAD

**Discussion by ishaantek** [vercel/community#5334] [Help]
### Summary

When adding a custom domain to my project, I only see the option to add an A Record. I don't see the nameservers tab. Can anyone help me? Thanks!

![image](https://github.com/vercel/community/assets/70829999/8145ce86-3a93-48a1-8da5-814a8fbfb4f9)

### Steps to Reproduce

N/A

**Comment by JuanM04** [comment]
Same here, I'm trying to setup `jms.ar`, but I ran into the same issue.

**Reply by JuanM04** [reply]
Update: I've fixed it. I was using this domain with https://dub.co. It wasn't after I removed from their service that I the "Nameservers" tab appeared.
